pkg/helm/handlers/handlers.go

```diff
@@ -109,7 +117,7 @@ func (h *helmHandlers) HandleHelmList(user *auth.User, w http.ResponseWriter, r
 func (h *helmHandlers) HandleGetRelease(user *auth.User, w http.ResponseWriter, r *http.Request) {
 	queryParams := r.URL.Query()
 	ns := queryParams.Get("ns")
-	chartName := queryParams.Get("release_name")
+	chartName := queryParams.Get("name")
```

Comment:
did you upgrade doc to reflect this? This change is totally unrelated to the ticket, hence I would avoid it, and instead of that open another PR just for this.

Reply:
Docs has already been updated in this PR

Reply:
Reverted